client/app/(todos)/components/Alert.tsx

```diff
@@ -0,0 +1,33 @@
+import { FiAlertTriangle } from 'react-icons/fi'
+import { RouterLink } from '@/_components/ui'
+
+type AlertProps = {
+  mainMessage: string
+  linkHref: string
+  linkText: string
+  additionalText?: string
+}
+
+const Alert = ({
+  mainMessage,
+  linkHref,
+  linkText,
+  additionalText = '',
+}: AlertProps) => (
+  <div className='flex flex-row gap-6 rounded-sm border-[1px] border-red-400 bg-red-100 px-5 py-6 md:px-8'>
```

Comment:
I think for smartphones, it is too much gap. `gap-3` for sp?

Reply:
Fixed it!
https://github.com/HAK2024/fairy-share/pull/71/commits/3fb6fce2e7e765f07138238ee4aaf574d968c759
https://github.com/HAK2024/fairy-share/pull/71/commits/2398a2838cf9a13e1574f9df78857418e63208c8